src/Magnum/Math/Geometry/Distance.h

```diff
@@ -162,6 +162,34 @@ class Distance {
          * the square root.
          */
         template<class T> static T lineSegmentPointSquared(const Vector3<T>& a, const Vector3<T>& b, const Vector3<T>& point);
+
+        /**
+         * @brief Distance of point from plane, scaled by the length of the planes normal.
+         * @param point the point
+         * @param plane the plane
+         * @returns a value `> 0` if point is on positive half space,
+         *                  `< 0` if point is on negative half space,
+         *                  `0` if point is on the plane.
+         *
+         * More efficient than @ref pointPlane() when the planes normal is already
+         * normalized or when testing on which half space the point lies.
+         */
+        template<class T> static T pointPlane(const Vector3<T>& point, const Vector4<T>& plane) {
+            return (plane.xyz()*point).sum() + plane.w();
+        }
+
+        /**
+         * @brief Distance of point from plane.
+         * @param point Point
+         * @param plane Plane
+         * @returns `> 0` if point is in front of plane,
+         *          `< 0` if point is behind the plane,
+         *          `0` if point is on the plane.
+         */
```

Comment:
Besides the above I would reference the other function as a faster alternative to this one.